lib/vm/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 //! Runtime library support for Wasmer.
 
-#![deny(missing_docs, trivial_numeric_casts, unused_extern_crates)]
```

Comment:
🙀 

I think we should be testing this in CI, not in the crate unconditionally during development, but especially in `vm`, we need everything to be properly documented